### PR TITLE
Epic/31 movies

### PR DESCRIPTION
**Pull request by phpgeekfr**

### CODE REVIEW

File: libs/movie/src/lib/movie/form/form.component.ts [hunk 1]
```diff
@@ -28,7 +28,8 @@ export class FormComponent implements OnInit, OnDestroy {
   public movie: Movie;
   private isModifying = false;
   private activeId: string;
-
+  public fullScreenForm = 'hide';
```
Comment (GrandSchtroumpf) [inline libs/movie/src/lib/movie/form/form.component.ts]: use boolean

File: libs/movie/src/lib/movie/form/form.component.ts [hunk 2]
```diff
@@ -280,4 +279,11 @@ export class FormComponent implements OnInit, OnDestroy {
   }
 
 
+  public toggleFullScreen() {
```
Comment (GrandSchtroumpf) [inline libs/movie/src/lib/movie/form/form.component.ts]: Use ternary

File: libs/movie/src/lib/movie/list/list.component.html [hunk 3]
```diff
@@ -16,7 +16,7 @@ <h3 class="mat-h3">ADD A MOVIE</h3>
...
...ABghABghABghABghABghAEYoAQBGCABGCABGCABGCABGCABGCABGCABGCABGCABGCABGCABGCABGCABGCABGCABGCABGCABGCABGCABGCABGCABGCABGCABGCABGCABGCABGCABGCABGCABGCABGCABGCABGCABGCABGCABGCABGCABGCABGCABGCABGCABGCABGCABGCABGCABGCABGCABGCABGCABGCABGCIARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAmCEAGCEAGCEAGCEAGCEAGCEAGCEAGCEAGCEAGCEAGCEAGCEAGCEAGCEAGCEAGCEAGCEAGCEAGCEAGCEAGCEAGCEAGCEAGCEAGCEAGCEAGCEAGCEAGCEAGCEAGCEAGCEAGCEAGCEAGCEAGCEAGCEAGCEAGCEAGCEAGCEAGCEAGCEAGCEAGCEAGCEAGCEAGCEAGCEAGCEAGCEABghABghABghABghABghABghABghABghABghABghABghABghABghABghABghABghABghABghABghABghABghABghABghABghABghABghABghABghABghABghABghABghABghABghABghABghABghABghABghABghABghABghABghABghABghABghABghABghABghABghABghABghAEYIAEYIAEYIAEYIAEYIAEYIAEYIAEYIAEYIAEYIAEYIAEYIAEYIAEYIAEYIAEYIAEYIAEYIAEYIAEYIAEYIAEYIAEYIAEYIAEYIAEYIAEYIAEYIAEYIAEYIAEYIAEYIAEYIAEYIAEYIAEYIAEYIAEYIAEYIAEYIAEYIAEYIAEYIAEYIAEYIAEYIAEYIAEYIAEYIAEYIAEYIgBFKAIARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAoARAsAFCwJICIYtDQbwAAAAAElFTkSuQmCC"></ng-template>
```
Comment (GrandSchtroumpf) [inline libs/movie/src/lib/movie/list/list.component.html]: Look how to put the image in asset.

File: libs/movie/src/lib/movie/list/list.component.html [hunk 4]
```diff
@@ -44,6 +44,8 @@ <h3 class="mat-h3">ADD A MOVIE</h3>
             <mat-card-title *ngIf="movie.title">
               {{ movie.title.original }}
             </mat-card-title>
+            <mat-card-subtitle *ngIf="movie.productionYear">{{ movie.productionYear }}</mat-card-subtitle>
+            <mat-card-subtitle *ngIf="movie.directorName">{{ movie.directorName }}</mat-card-subtitle>
```
Comment (GrandSchtroumpf) [inline libs/movie/src/lib/movie/list/list.component.html]: `<mat-card-subtitle>{{ movie?.directorName }}</mat-card-subtitle>` ????

File: libs/movie/src/lib/movie/list/list.component.ts [hunk 5]
```diff
@@ -36,7 +36,7 @@ export class ListComponent implements OnInit, OnDestroy {
     this.isAlive = false;
   }
   public addNewMovie(orgId, orgName) {
-    this.dialog.open(TitleFormComponent, { data: { org: { id: orgId, name: orgName } } });
+    this.dialog.open(TitleFormComponent, { data: { id: orgId, name: orgName } });
```
Comment (GrandSchtroumpf) [inline libs/movie/src/lib/movie/list/list.component.ts]:
```typescript
addNewMovie(org: { id: string, name: string}) {
   this.dialog.open(TitleFormComponent, { data: org });
}
```

File: libs/movie/src/lib/movie/list/list.component.html (hunk 4 above)
Comment (phpgeekfr) [reply libs/movie/src/lib/movie/list/list.component.html]: This proposition shows empty `<mat-card-subtitle>` tag.

**Pull request merged**